plugin.rb

```diff
@@ -25,7 +25,6 @@
 add_admin_route 'explorer.title', 'explorer'
 
 module ::DataExplorer
-  QUERY_RESULT_DEFAULT_LIMIT = 1000
   QUERY_RESULT_MAX_LIMIT = 10000
```

Comment:
Do we still need to keep this constant around?

Reply:
Ahh it looks we do. Perhaps we can add a comment here to make sure this value and the site setting's value should always be the same?

Reply:
Yes it's still used for CSV format:

https://github.com/discourse/discourse-data-explorer/blob/015a35350e6f7375f89b6efeae470edc0df13deb/app/controllers/data_explorer/query_controller.rb#L137-L145

Reply:
Your second comment hadn't loaded for me before I sent that, sorry. I will add the comment.